platform_versioned_docs/version-23.3.0/enterprise/configuration/ssl_tls.mdx

```diff
@@ -15,6 +15,12 @@ Use [Amazon Certificate Manager](https://aws.amazon.com/certificate-manager/) (A
 
 - If you don't have an existing SSL certificate, see [Issuing and managing certificates](https://docs.aws.amazon.com/acm/latest/userguide/gs.html).
 
+## Azure deployments: Use Application Gateway for TLS termination and load balancing
```

Comment:
Not sure if we want to include this for now. We have an outstanding issue with using the application gateway here: https://github.com/seqeralabs/tower-backlog/issues/1244

Reply:
Thanks. Leaving out for now.